CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Cromwell Change Log
 
+## 26
+
+* Added a configuration knob under `system.io` to throttle the number of I/O queries that Cromwell makes.
```

Comment:
hah it is. well it *can* be a bad word but that's not what I meant. It just sounds less polished :)